### PR TITLE
+pa_solution.0.6

### PR DESCRIPTION
**Pull request by cakeplus**

### CODE REVIEW

**Comment by cakeplus** [comment]
I have updated `opam` to use the latest specification.

That linting bot is very cool. :+1:

File: packages/pa_solution/pa_solution.0.6/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "Max Mouratov <mmouratov@gmail.com>"
+author: "Max Mouratov <mmouratov@gmail.com>"
+homepage: "https://github.com/cakeplus/pa_solution"
+license: "LGPL-2.1 with OCaml linking exception"
+dev-repo: "https://github.com/cakeplus/pa_solution"
```
Comment (dsheets) [inline packages/pa_solution/pa_solution.0.6/opam]: This needs `.git` appended for `opam` to understand the VCS protocol to use.

Comment (cakeplus) [inline packages/pa_solution/pa_solution.0.6/opam]: Fixed it. Thank you!

Perhaps the linting tool should also check for that?

Comment (dsheets) [inline packages/pa_solution/pa_solution.0.6/opam]: ocaml/opam#2260 fixed it but it hasn't been released, yet, afaict.

**Pull request merged**